AutomatticTracks/build.gradle

```diff
@@ -38,16 +38,24 @@ android {
     }
 }
 
-
-publish {
-    userOrg = 'automattic'
-    groupId = 'com.automattic'
-    uploadName = 'tracks'
-    artifactId = 'tracks'
-    desc = 'Android client for Nosara tracks (event tracking and analytics)'
-    publishVersion = android.defaultConfig.versionName
-    licences = ['MIT']
-    website = 'https://github.com/Automattic/Automattic-Tracks-Android'
-    dryRun = 'false'
-    autoPublish = 'true'
+afterEvaluate {
+    publishing {
+        publications {
+            maven(MavenPublication) {
+                artifactId 'Automattic-Tracks-Android'
+                artifact("$buildDir/outputs/aar/AutomatticTracks-debug.aar")
+                description 'Android client for Nosara tracks (event tracking and analytics)'
+                groupId 'com.github.Automattic'
```

Comment:
Changing `artifactId` and `groupId` to match convention forced by `Jitpack` so using local maven will require only changing version, not the whole dependency address.

Local maven artifactory might not be that useful after migration to composite builds mentioned [here](https://github.com/Automattic/Automattic-Tracks-Android/pull/51/files#r582846727) but for now I think it adds a convenient way to work with `Tracks` locally.

Reply:
I don't think we should be using local maven. I'd rather just make it possible to use composite builds with this library instead as that should be very straightforward for this library. I think we should just stick to `com.automattic` for the `groupId`.